package.json

```diff
@@ -63,19 +65,23 @@
         "grunt-contrib-watch": "^1.1.0",
         "grunt-exec": "^3.0.0",
         "grunt-sass": "^3.0.2",
+        "identity-obj-proxy": "^3.0.0",
         "jest": "^24.7.1",
         "jest-circus": "^24.7.1",
         "jest-junit": "^6.3.0",
         "license-check-and-add": "2.3.6",
         "make-dir": "^1.3.0",
+        "mini-css-extract-plugin": "^0.6.0",
         "node-sass": "^4.12.0",
         "npm-run-all": "^4.1.5",
         "preprocess": "^3.1.0",
         "prettier": "^1.17.0",
         "puppeteer": "^1.15.0",
+        "sass-loader": "^7.1.0",
         "script-loader": "0.7.2",
         "serve-static": "^1.13.2",
         "source-map-loader": "^0.2.4",
+        "style-loader": "^0.23.1",
```

Comment:
I'm not seeing where this dep is used; is it leftover from other experimenting?

Reply:
Correct!